framework/ramcache.py

```diff
@@ -153,11 +156,13 @@ class SharedInvalidateParent(ndb.Model):
 class SharedInvalidate(ndb.Model):
 
   PARENT_ENTITY_ID = 1234
-  PARENT_KEY = ndb.Key('SharedInvalidateParent', PARENT_ENTITY_ID)
   SINGLETON_ENTITY_ID = 5678
-  SINGLETON_KEY = ndb.Key(
+  with client.context():
+    PARENT_KEY = ndb.Key('SharedInvalidateParent', PARENT_ENTITY_ID)
+    SINGLETON_KEY = ndb.Key(
       'SharedInvalidateParent', PARENT_ENTITY_ID,
       'SharedInvalidate', SINGLETON_ENTITY_ID)
+  
```

Comment:
See [this](https://github.com/GoogleChrome/chromium-dashboard/pull/1351#discussion_r645886479) link

Reply:
That makes sense.